ci_build/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Image for launching TensorFlow Haskell tests.
+FROM tensorflow/tensorflow:nightly-devel
+# gcr.io/tensorflow/tensorflow:latest-devel
+MAINTAINER TensorFlow authors <tensorflow-haskell@googlegroups.com>
+
+RUN \
+    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 575159689BEFB442 && \
+    echo 'deb http://download.fpcomplete.com/ubuntu trusty main'| tee /etc/apt/sources.list.d/fpco.list
+
+RUN apt-get update
+
+RUN apt-get install -y \
+        # Avoids /usr/bin/ld: cannot find -ltinfo
+        libncurses5-dev \
+        # Makes stack viable in the container
+        libgmp-dev \
+        libcurl3-dev \
+        stack
+
+# The build context directory is the top of the tensorflow-haskell
+# tree.
+ADD . /tfhs
+
+WORKDIR /tfhs
+
+RUN git submodule init && git submodule update
+
+# Installs protoc and the libraries.
+RUN \
+    cd third_party/tensorflow && \
```

Comment:
The goal was to make the build more hermetic as tensorflow submodule checkout is pinned.